helm_deploy/values-preprod.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# Per environment values which override defaults in hmpps-incentives-api/values.yaml
+
+generic-service:
+  replicaCount: 4
+
+  ingress:
+    host: incentives-api-preprod.hmpps.service.justice.gov.uk
+    contextColour: green
+
+  env:
+    APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.dev.json
```

Comment:
Is `preprod` sharing this configuration file with `dev`?

Reply:
yes